### PR TITLE
Improved Shortcut Scheme

### LINKED ISSUE #492: Improve Shortcut Scheme
Currently the menu shortcuts only support languages in Latin letters. My plan is to make all the shortcuts separated by moving them to brackets, like changing `&amp;Pause` to `Pause(&amp;P)`. In this case, all languages will be supported. Before the pull request is made, I'm wondering if anyone has better idea about that.

### PR DESCRIPTION
**Pull request by zo-shin**
Made all the shortcuts separated by moving them to brackets and other minor changes.
close #492

### CODE REVIEW

File: src/gui/MainWidget.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```
Comment (zaps166) [inline src/gui/MainWidget.cpp:1]: Please don't add UTF-8 BOM.

**Review by zaps166 — CHANGES_REQUESTED** [review]

**Comment by zaps166** [comment]
ehh doesn't compile

**Comment by zaps166** [comment]
I fixed XML files, but it looks very bad:

Screenshot_20220425_004045

Reverted.

**Comment by zo-shin** [comment]
What do you think is the problem? Is it OK if I just make changes to the language TS files only?

**Comment by zaps166** [comment]
Changes to TS files are not a problem. I uploaded a screenshot of menu bar - it looks vary bad with brackets at the end: `Help(H)`, etc... Same with menu items.

**Comment by zo-shin** [comment]
If that looks bad, do you have any better solution about the issue? It seems a common way to deal with shortcuts.

**Comment by zaps166** [comment]
Maybe add it only for Chinese translations?

**Comment by zo-shin** [comment]
All the languages should be solved in the same way, not only Chinese.

**Comment by zaps166** [comment]
> All the languages should be solved in the same way, not only Chinese.

Chinese, Japanese. Example:


- QtCreator Chinese
Screenshot_20220430_123356

- QtCreator English
Screenshot_20220430_123427

**Pull request merged**